src/views/RecIM/components/Forms/EditMatchStatsForm/index.js

```diff
@@ -0,0 +1,245 @@
+import { Grid } from '@mui/material';
+import { useState, useMemo, useEffect } from 'react';
+import GordonDialogBox from 'components/GordonDialogBox';
+import { InformationField } from '../components/InformationField';
+import { ConfirmationWindowHeader } from '../components/ConfirmationHeader';
+import { ConfirmationRow } from '../components/ConfirmationRow';
+import { ContentCard } from '../components/ContentCard';
+import GordonLoader from 'components/Loader';
+import { getMatchStatusTypes, updateMatchStats } from 'services/recim/match';
+
+const EditMatchStatsForm = ({
+  teamMatchHistory,
+  closeWithSnackbar,
+  openEditMatchStatsForm,
+  setOpenEditMatchStatsForm,
+}) => {
+  const [errorStatus, setErrorStatus] = useState({
+    Score: false,
+    StatusID: false,
+  });
+
+  const [loading, setLoading] = useState(true);
+  const [matchStatus, setMatchStatus] = useState([]);
+
+  useEffect(() => {
+    const loadData = async () => {
+      setLoading(true);
+      setMatchStatus(await getMatchStatusTypes());
+      setLoading(false);
+    };
+    loadData();
+  }, []);
+
+  const createMatchStatsField = [
+    {
+      label: 'Score',
+      name: 'Score',
+      type: 'text',
+      error: errorStatus.Name,
+      helperText: '*Required',
+    },
+    {
+      label: 'Sportsmanship',
+      name: 'Sportsmanship',
+      type: 'text',
+      error: errorStatus.Name,
+      helperText: '*Required',
+    },
+    {
+      label: 'Status',
+      name: 'StatusID',
+      type: 'select',
+      menuItems: matchStatus.map((type) => {
+        return type.Description;
+      }),
+      error: errorStatus.Name,
+      helperText: '*Required',
+    },
+  ];
```

Comment:
Your error handling does not work? 
errorStatus.Name does not exist, you have:
`const [errorStatus, setErrorStatus] = useState({
    Score: false,
    StatusID: false,
  });`